torchvision/prototype/datasets/utils/_dataset.py

```diff
@@ -28,6 +28,7 @@
 class DatasetType(enum.Enum):
     RAW = enum.auto()
     IMAGE = enum.auto()
+    VIDEO = enum.auto()
```

Comment:
We also need to update 

https://github.com/pytorch/vision/blob/0817f7f5983b05fdc34e06476ca270e2168c57b8/torchvision/prototype/datasets/_api.py#L54-L57